images/rootfs.yml.in

```diff
@@ -25,8 +25,14 @@ onboot:
     capabilities:
       - CAP_SYS_ADMIN
       - CAP_NET_ADMIN
+  # Interactive installer must preseed storage-init
+  # If you change the order of interactive-installer don't forget to
+  # change path to the "002-interactive-installer" in "pkg/mkimage-raw-efi/config"
+  - name: interactive-installer
+    image: INTERACTIVE_INSTALLER_TAG
+    command: ["/bin/sh", "-c", "/sbin/run-installer.sh </dev/console >/dev/console 2>&1"]
```

Comment:
I don't quite get how this works. Does this mean the interactive installer will run on every single boot? This is in the `rootfs.yml.in`, which makes it part of every rootfs in the `onboot` serial path. Is that intended? Don't we usually have distinct installer images and running rootfs images?

Reply:
We want this because we will include a maintenance mode afterwards. So, we need to have it running for each boot.

Reply:
@jsfakian @rucoder do you have a drawing of what the pieces will when we are done (with "maintenance mode" aka "network config repair", and "post-install network config", and the interactive installer)?

We need to make sure we're all on the same page of how those pieces will interact.